### PR TITLE
Merge Linaro changes into the master branch

### PR DESCRIPTION
**Pull request by BaptisteGerondeau**
This set of 8 patches takes the master branch from a non installing state in our lab, to a fully working one (barring Linaro Infrastructure specific changes, and the 1.3.5 to 1.3.6 patch(es)).

### CODE REVIEW

File: site.yml [hunk 1]
```diff
@@ -85,7 +86,6 @@
     - third-party-libs-gnu7
     - third-party-libs-intel
     - imb
-    - slurm-client
     - warewulf-final
-    # - booting-computing-nodes/
-    - final
+    # - booting-computing-nodes
+    - final
```
Comment (rengolin) [inline site.yml]: unnecessary extra space

**Comment by rengolin** [comment]
These changes look good to me, and they make a lot of sense. 

The re-ordering is necessary to make sure changes are done before services are restarted.

The splitting of init/final is necessary when nodes are provisioned in parallel (not through Warewulf).

@takekato can you please try these in your branch and make comments / approve them, so we can merge to master?

Meanwhile, we'll try them on our infra and, if it works the same, we'll merge into Linaro's branch.

**Comment by ghost** [comment]
> @takekato can you please try these in your branch and make comments / approve them, so we can merge to master?

Sure.
I'll check these changes in our environment after Baptiste fixes three things(?) Renato commented as above.

**Comment by BaptisteGerondeau** [comment]
Whitespace issue fixed !

**Comment by BaptisteGerondeau** [comment]
@takekato A working "dev-complete" branch with the above patches, and [this one (1.3.5 to 1.3.6)](https://github.com/Linaro/ansible-playbook-for-ohpc/commit/065689743549c5d5964769efa3118f1b665ba28e) can be found here : https://github.com/Linaro/ansible-playbook-for-ohpc/tree/dev-complete

**Pull request closed without merging**